src/main/java/com/salesforce/dynamodbv2/mt/mappers/MTAmazonDynamoDBByAccount.java

```diff
@@ -21,30 +21,72 @@
 
 /**
  * @author msgroi
- *
+ * <p>
  * Allows for dividing tenants into different AWS accounts.  To use, pass in AmazonDynamoDBClientBuilder and MTAccountCredentialsMapper.
  * At run-time, a String representing the multi-tenant will be passed to your credentials mapper, allowing you map
  * the context to different AWS credentials implementations.
- *
+ * <p>
  * MTAmazonDynamoDBByAccount does not support delegating to a mapper and therefore must always be at the end of the chain when it is used.
- *
+ * <p>
  * To use, call the static builder() method.  The following parameters are required ...
  * - a multi-tenant context
  * - a MTAccountCredentialsMapper implementation that maps context to AWSCredentialsProvider's
- *
+ * <p>
  * Supported:
- *    - methods: create|describe|delete Table, get|putItem, scan|query
+ * - methods: create|describe|delete Table, get|putItem, scan|query
  */
 public class MTAmazonDynamoDBByAccount extends MTAmazonDynamoDBBase {
 
-    public interface MTAccountCredentialsMapper {
-        AWSCredentialsProvider getAWSCredentialsProvider(String context);
+    private final MTAccountMapper accountMapper;
```

Comment:
what happened here (and all over this class)? method (and inner class) reordering?